development/build/README.md

```diff
@@ -26,38 +26,4 @@ Source file bundling tasks are implemented in the [`./development/build/scripts.
 
 ## Usage
 
-```text
```

Comment:
At some point I would like to explore bringing this back, using a tool that can automatically keep this in-sync with the CLI. No point in maintaining it twice in the meantime though